dotnet/src/Extensions/Extensions.UnitTests/Planning/ActionPlanner/ActionPlannerTests.cs

```diff
@@ -78,7 +78,7 @@ public async Task InvalidJsonThrowsAsync()
         var planner = new Microsoft.SemanticKernel.Planning.ActionPlanner(kernel.Object);
 
         // Act & Assert
-        await Assert.ThrowsAsync<PlanningException>(async () => await planner.CreatePlanAsync("goal"));
+        await Assert.ThrowsAsync<SKException>(async () => await planner.CreatePlanAsync("goal"));
```

Comment:
I believe this might work, without the async, same for others.

```suggestion
        await Assert.ThrowsAsync<SKException>(() => planner.CreatePlanAsync("goal"));
```